packages/core-flows/src/inventory/steps/validate-inventory-items-for-create.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ContainerRegistrationKeys,
+  MedusaError,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+import { InventoryNext } from "@medusajs/types"
+
+export const validateInventoryItemsForCreateStepId =
+  "validate-inventory-items-for-create-step"
+export const validateInventoryItemsForCreate = createStep(
+  validateInventoryItemsForCreateStepId,
+  async (
+    input: {
+      inventoryItem: InventoryNext.CreateInventoryItemInput
+      tag?: string
+    }[],
+    { container }
+  ) => {
+    const remoteQuery = container.resolve(
+      ContainerRegistrationKeys.REMOTE_QUERY
+    )
+
+    const query = remoteQueryObjectFromString({
+      entryPoint: "product_variant_inventory_item",
```

Comment:
todo: this string if someone is not aware of the link seems to be getting from nowhere, should we use `LINKS.ProductVariantInventoryItem`

Reply:
well, we should not use the remote query here anyways right, we should just use the link module service to list the links? 😉